### PR TITLE
Display image for image posts (detail view)

### PR DESCRIPTION
**Pull request by jmmxp**
I check contentType and if it's any of the image types, I use an `img` tag instead of a `p` tag to display the content.

Also made sure to add 'content' as a string field on the Post class, something we missed earlier.

image

### CODE REVIEW

File: front-end/src/components/PostDetailModal.tsx [hunk 1]
```diff
@@ -10,13 +11,23 @@ interface Props {
 
 export default function PostListItem(props:Props) {
   const post: Post = props.post;
+
+  function Content() {
```
Comment (ChrisChrisLoLo) [inline front-end/src/components/PostDetailModal.tsx:15]: I think it's pretty clever to use functions like this over what I'm used to, which is just changing around scoped variables 👍

**Review by ChrisChrisLoLo — APPROVED** [review]
LGTM!

Does the backend currently serialize `content`, or does that need to be added in another PR?

File: front-end/src/components/PostDetailModal.tsx (hunk 1 above)
Comment (jmmxp) [reply front-end/src/components/PostDetailModal.tsx:15]: Good to know! I feel like I have no idea what I'm doing when it comes to React haha

**Comment by jmmxp** [comment]
It's already being serialized, so we're good to go :)

**Comment by ChrisChrisLoLo** [comment]
Cool. Good catch, I must have missed that field in my last PR

**Pull request merged**